src/action.py

```diff
@@ -2,35 +2,38 @@
 import os
 
 from main import shared_main
+from git import get_changed_jsons
 from filemanager import FileManager
 
 def main():
     # Argument parsing
     parser = argparse.ArgumentParser()
-    parser.add_argument('filepath', help='Path of JSON output file to parse')
+    parser.add_argument('jsondir', help='Path of JSON directory to parse files from')
     parser.add_argument('hashref', help='Hash value of commit to compare against (reference commit)')
     parser.add_argument('hashcomp', help='Hash value of commit to compare')
     parser.add_argument('output_path', help='Path of folder to output the generated page')
     args = parser.parse_args()
 
-    filepath = os.path.abspath(args.filepath)
+    jsondir = os.path.abspath(args.jsondir)
     hashref = args.hashref
     hashcomp = args.hashcomp
     output_path = os.path.abspath(args.output_path)
 
-    print(f'Your filepath: {filepath}')
+    print(f'Your JSON directory: {jsondir}')
     print(f'Your reference commit hash: {hashref}')
     print(f'Your commit hash to compare: {hashcomp}')
     print('-----------------------------------------------')
 
-    # File parsing
-    ref_file = FileManager(filepath, hashref)
-    comp_file = FileManager(filepath, hashcomp)
-    ref_content = ref_file.parse()
-    comp_content = comp_file.parse()
+    changed_jsons = get_changed_jsons(jsondir, hashcomp)
+    for changed_json in changed_jsons:
```

Comment:
Is `changed_json` a file path? If so, maybe keep `filepath` (or `file_path`) for clarity.

Reply:
(This would look better in PR reviews too 😉)